### ISSUE TITLE
Review our ActiveJob integration

### ISSUE THREAD

**Issue by tombruijn** [appsignal/appsignal-ruby#343]
Review our ActiveJob integration after #345.

In delayed job were all reported with the same name: `ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper`

maybe the same assumption was made in Sidekiq, so let's review other AJ integrations and see it they still work.

Bonus: Add examples app for each variation to https://github.com/appsignal/appsignal-examples

**Comment by tombruijn** [comment]
https://app.intercom.io/a/apps/yzor8gyw/respond/inbox/conversation/11314730559

**Comment by tombruijn** [comment]
https://app.intercom.io/a/apps/yzor8gyw/respond/inbox/conversation/14359602324

**Comment by tombruijn** [comment]
Duplicate of https://github.com/appsignal/appsignal-ruby/issues/330
